### PR TITLE
[DX-760] Patch Dev Support Release Summary With Link To Helm Charts

### PR DESCRIPTION
**Pull request by dcs3spp**
Patch Dev Support Release Summary With Link To Helm Charts

[preview](https://deploy-preview-3422--tyk-docs.netlify.app/docs/nightly/developer-support/tyk-release-summary/overview)

[DX-760](https://tyktech.atlassian.net/browse/DX-760)

[DX-760]: https://tyktech.atlassian.net/browse/DX-760?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: tyk-docs/content/developer-support/tyk-release-summary/overview.md [hunk 1]
```diff
@@ -8,3 +8,5 @@ aliases:
 Since release 5.2 separate release notes are available for [Tyk Dashboard]({{< ref "product-stack/tyk-dashboard/release-notes/overview" >}}) and [Tyk Gateway]({{< ref "product-stack/tyk-gateway/release-notes/overview" >}}).
 
 Prior to release 5.2 release notes for Tyk Dashboard and Tyk Gateway were bundled into the same release note.This section provides access to these older release notes.
+
+Release notes for our helm charts can be found at our [GitHub repository](https://github.com/TykTechnologies/tyk-charts/releases.)
```
Comment (carolwkc) [inline tyk-docs/content/developer-support/tyk-release-summary/overview.md]: Extra dot in the link
```suggestion
Release notes for our helm charts can be found at our [GitHub repository](https://github.com/TykTechnologies/tyk-charts/releases)
```

**Review by carolwkc — CHANGES_REQUESTED** [review]
Thank you @dcs3spp , just correcting the hyperlink

**Comment by dcs3spp** [comment]
> Thank you @dcs3spp , just correcting the hyperlink

Many thanks @caroltyk, have requested re-review. If it is ok can you approve and I will merge and release.

**Review by carolwkc — APPROVED** [review]

**Pull request merged**